docs/source/quickstart.rst

```diff
@@ -1,8 +1,53 @@
 Getting started
 ============================================
 
-with simplay
-------------
+
+.. _installation:
+
+Install SimPlay
+---------------
+
+To use Simplay, first decide how you want to use it.
+Three components exist:
+
+  - `SimPlay-Jupyter <https://pypi.python.org/pypi/simplay-jupyter>`_ (the python package allowing you to view and enhance any SimPy Simulation within a jupyter notebook)
+  - `SimPlay <https://pypi.python.org/pypi/simplay>`_ (the python library allowing you to generate the SimulationData)
+  - `SimPlay-Web <https://www.npmjs.com/package/simplay-web>`_ (the npm packages allowing you to render the SimulationData)
+
+To install the jupyter extension, execute the following command:
+
+.. note::
+   This guide assumes you have a working JupyterLab installation.
+   The minimum version required is 3.0.
+
+.. code-block:: console
+
+   pip install simplay-jupyter
+
+.. note::
+   After installing simplay-jupyter, JupyterLab needs to be restarted.
+
+The following installation steps are only necessary, if you don't want to run within JupyterLab,
+otherwise continue with :ref:`AddVisualInformation`.
```

Comment:
```suggestion
The following installation steps are only necessary if you don't want to run within JupyterLab.
Otherwise continue with :ref:`AddVisualInformation`.
```